src/main/java/King.java

```diff
@@ -0,0 +1,42 @@
+public class King implements Calculator {
+
+    @Override
```

Comment:
This may cause an autograde error, aside from the ones caused from unimplemented classes.